automatic/minishift/tools/chocolateyInstall.ps1

```diff
@@ -11,4 +11,5 @@ $packageArgs = @{
   FileFullPath64 = Get-Item $toolsPath\minishift-*-windows-amd64.zip
```

Comment:
Can we please get rid of the globbing resolving of paths to files, and rather use the full path to the file (can be updated in `update.ps1`).

This has caused issues quite a few times in the past.